### PR TITLE
Update and rename languages-for-voicemail-greetings-and-messages.md t…

### PR DESCRIPTION
**Pull request by CLYVR**
…o languages-for-voicemail-greetings-and-messages-teams.md

Update to Teams voicemail languages.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy is fine, but the table does not render correctly and needs to be fixed (see rich diff view).

File: Teams/manage-voicemail-policies.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+audience: Admin
+appliesto: 
+  - Skype for Business
+  - Microsoft Teams
+localization_priority: Normal
+f1.keywords:
+- CSH
+ms.custom: 
+  - Phone System
+description: "Manage Voicemail Policies for your users. "
```
Comment (JohanFreelancer9) [inline Teams/manage-voicemail-policies.md:24]:
```suggestion
description: "Manage Voicemail Policies for your users. "
```
```suggestion
description: "Manage Voicemail Policies for your users."
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@CLYVR - Just one small edit. The copy and table are fine now but I will do a final copy edit when the status has been changed from "Draft". Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by CLYVR** [comment]
@JohanFreelancer9 What remains to be done in order to get these merged to live?  Thanks!

**Comment by JohanFreelancer9** [comment]
@scanum?

**Pull request merged**